### DISCUSSION TITLE
Trabalho remoto: internacional, remuneração e experiências

### DISCUSSION THREAD

**Discussion by willycamargo** [frontendbr/forum#1510] [Discussão]
## Trabalho remoto
Thread para portar discussão interessante que nasceu [no lugar errado](https://github.com/frontendbr/vagas/issues/2290).

Para contextualizar: um usuário reclamou da vaga ter uma remuneração teto de R$8.000 para trabalho remoto internacional (PJ) com requisito de inglês fluente e 2 anos de experiência com React. Os comentários desse usuário foram removidos da thread, provavelmente por ele mesmo.

Em contrapartida deixei minha opinião, explicando que acho comum vagas remotas com salários baixos, mas acredito que elas não são sustentáveis, fazendo com que o usuário procure outra oportunidade após um tempo caso não recebe um aumento dentro da própria empresa.

Pois então: gostaria de saber a **opinião** de vocês, principalmente de quem teve/tem experiência, sobre o mercado de trabalho remoto internacional e remuneração.

Vamos tentar falar de valores R$ sem tabu e de forma objetiva. Quem quiser trazer dados externos para aquecer o debate, fique a vontade.

### Pontos que acho interessante analisar antes de soltar opiniões

1. O que leva um profissional a buscar um trabalho remoto?
2. O que leva uma empresa a buscar desenvolvedores remotos?
3. O inglês é um diferencial que deveria potencializar a remuneração? Por quê?
4. No contexto de trabalho remoto internacional, nós merecemos receber mais do que no mercado local? Por quê?
5. Vamos tentar não levar a discussão para o campo político.

**Comment by willycamargo** [comment]
Moro em uma cidade do interior do Paraná. Por motivos pessoais é inviável para mim ir morar em uma grande capital. Os salários por aqui dificilmente passam de 4mil (PJ), o que me fez procurar por oportunidades remotas.

Trabalhei em duas empresas remotamente, sempre como PJ, e em nenhuma delas a remuneração passou de 10mil mensais (bruto, algo em torno de 8mil liquido).

Deixando **minha opinião**:

Acredito que a remuneração de R$8.000/mês é baixa para um profissional pleno/senior com 2 anos de experiência em uma tecnologia específica e que terá a responsabilidade de:
* Abrir empresa (pagar taxas + impostos)
* Adquirir e fazer manutenção em equipamento próprio
* Garantir disponibilidade de hardware, internet, energia e infraestrutura em geral

Mas não acho que trabalho remoto internacional seja o ponto principal para a potencialização da remuneração. O trabalho final ainda será planejar, estruturar, desenvolver e testar aplicações. O idioma é apenas a porta de entrada para mais oportunidades.

Em contraponto, acredito que oportunidades remotas com "baixa remuneração" é comum no mercado. Sempre presenciei alguns casos e tem sido mais frequente ultimamente, e acredito que faça sentido porque:

1. As empresas acabam atingindo as pessoas que preferem morar em cidades mais afastadas (e consequentemente com menos oportunidades).
2. É uma boa porta de entrada para quem ainda não tem experiência com trabalho remoto.
3. Após algum tempo a pessoa percebe que é um pagamento abaixo para a atividade e renegocia ou então arruma outro trabalho remoto para quem pague mais.*

*Acredito que empresas que possuem esse salário como entrada e não renegociam o salário acabam tendo uma rotatividade bem alta. Presenciei em algumas experiências.

**Comment by gabriel-hahn** [comment]
Acredito que R$ 8.000 para uma vaga remota internacional é um valor aceitável. Sim, precisa de inglês e ter experiência em React neste caso, mas como já mencionado, se o desenvolvedor for realmente bom e a empresa não ajustar o salário, ele irá procurar outras oportunidades e quem perde é a empresa principalmente. 
Tudo depende muito do padrão de vida da pessoa e região que mora. Alguém que mora em São Paulo capital tem um custo de vida muito maior de alguém que mora no interior. Há muitas variáveis que definem se um salário é bom ou não.

No caso da vaga informada, talvez seja interessante descrever um pouco mais sobre a posição para entender melhor o nível de "senioridade" exigido. Mesmo com as exigências de um profissional qualificado, vale lembrar que se trata de uma vaga que paga 8x o salário mínimo brasileiro e conforme minhas rápidas pesquisas, é um valor dentro do salário mínimo inglês. Um desenvolvedor que aplicasse para uma vaga remota em Portugal por exemplo em que o salário mínimo é €600.00, talvez não recebesse esses 8 mil reais, variando bastante de país para país (já que o desenvolvedor iria ganhar mais que o salário mínimo, mas serve como base em alguns cálculos).
Sobre o inglês nesse caso específico de uma empresa inglesa, é algo mínimo exigido para se comunicar, não é um "plus", temos que entender o lado da empresa também.

Onde eu moro no interior de SC, 8 mil por mês é um salário muito bom (não estou considerando a competência técnica necessária), dificilmente um desenvolvedor ganha esse salário aqui e o custo de vida é relativamente baixo. Porém um dev em SP pode não concordar comigo hehe

**Edit 28/12/2020:**
- Meu primeiro salário como desenvolvedor e de muita gente que eu conheço foi bem pouco acima de um salário mínimo, então sim, dependendo de onde você mora e o perfil de empresa que você trabalha, isso é uma realidade. Só utilizei como base para exemplificar.
- Para muitas empresas (não é meu pensamento), faz muito sentido pagar menos para um desenvolvedor que mora no exterior do que aquele dev que vai trabalhar presencialmente no escritório e possui algumas vantagens de comunicação por causa disso, na visão deles. Sim, receber um pouco a menos do salário padrão do local em que a empresa reside é normal, mas nem por isso pode ser um salário ruim para você. Existem outras variáveis, cada caso é um caso, mas de maneira geral as empresas que procuram se aproveitar demais dos desenvolvedores que vivem em outros países, também tem uma rotatividade altíssima.
- O cálculo não é somente ver qual o salário médio no país e pedir algo próximo desse valor. Já trabalhei na Alemanha e eu pagava em torno de 40% de impostos (mas olhando no contrato, meu salário era bem **legal** e correspondia ao que aparecia em sites como o Glassdoor, mas eu recebia quase metade dele líquido somente). Ao mesmo tempo que isso pode dar brechas para negociações de um salário líquido maior para você remotamente, também pode ser um dos principais motivos de uma empresa buscar devs fora, pois não precisa pagar impostos altíssimos (usando o exemplo da Alemanha ainda, não necessariamente é a realidade de todos países, mas acho que deu para pegar a ideia 😃 ).
- Eu disse que era um valor aceitável no meu ponto de vista, mas não que era um salário maravilhoso. Obviamente, um desenvolvedor com experiência, inglês afiado e boa comunicação tem muito mais oportunidades para receber salários maiores que o ofertado nessa vaga de exemplo trabalhando para empresas do exterior. Essa vaga pede 2 anos de experiência, não estamos falando de _senior_ aqui a nível de vaga, então 8 mil reais é um valor bem legal e aceitável para boa parte do nosso país, creio eu, principalmente como porta de entrada para trabalho com empresas do exterior. Depois, provavelmente o dev vai buscar outras oportunidades se a remuneração não for mais compatível, conforme ganha experiência 👍🏼 .

**Comment by bernardodiasc** [comment]
A vaga oferece um salário mínimo Londrino, não tem nada de errado se as expectativas forem compatíveis com a oferta. Não acho que um profissional senior toparia a não ser que estivesse desesperado digamos usando essa oportunidade como uma etapa para explorar novos horizontes, mas de fato pode ser uma oportunidade incrível para outros níveis técnicos.

**Comment by guibfo** [comment]
**Desculpa a wall-of-text** 

Eu particularmente acho desrespeitoso empresas que usam do trabalho remoto para oferecer salários muito abaixo do mercado delas, mas por outro lado, eu entendo o lado delas, e isso é uma prática muito comum, são raríssimas as empresas que oferecem trabalho remoto e pagam o valor de mercado de onde elas atuam. Você como Dev vai achar ruim a empresa fazer isso, você como empresa vai achar lindo contratar Indianos, Brasileiros, Argentinos, pagando uma fração do valor que gastaria com um dev local.

Desenvolvedores não ganham salário mínimo, não faz sentido tomar o salário mínimo de qualquer lugar que seja como base, desenvolvedores ganham salários acima da média em provavelmente qualquer país do mundo, em Londres por exemplo a média salarial fazendo uma busca rápida no google ([Fonte](https://www.adzuna.co.uk/jobs/salaries/london/front-end-developer)) é de £42.2k, isso da por volta de £3.5k mensais numa conta simples de dividir por 12 (em Portugal por exemplo o salário anual contam 14 salários se não me engano), que daria em conversão direta ~R$17.1k, mas como eu falei, desenvolvedores ganham geralmente acima da média, na mesma página podemos ver que a média em Londres para Frontend devs é £66k, ou seja, ~R$26.8k em conversão direta.

Veja que mesmo que o número dessa fonte esteja inflado, ainda assim temos uma margem bem grande entre os valores. Teoricamente para a vaga discutido em questão, o salário está em média 3x abaixo do que "deveria", o que não quer dizer que seja um salário ruim, mas acredito que aconteça exatamente o que o @willycamargo disse, o cara pega experiência, fica atrativo para outras empresas e vaza, mas isso acontece com empregos alocados também.

Não acho que discutir, ou questionar, o salário oferecido seja criar polêmica, quem postou a vaga pode simplesmente responder que não pode pagar salários compatíveis com o que pagaria para desenvolvedores locais, ou qualquer outra justificativa, ou nem justificar se não quiser, e pronto, cada um sabe onde o calo aperta, seja a empresa, quanto quem vai se candidatar

Agora pra responder especificamente os pontos levantados na post:

1. O que leva um profissional a buscar um trabalho remoto?
Pra mim a grande questão é ganho de tempo e liberdade (que vem de mãos dadas com a responsabilidade), não passar por estresses do dia-a-dia com trânsito e transporte público, deslocamento, além de dependendo da empresa você tem liberdade de horários, podendo ir pra academia no meio da tarde que está completamente vazia e voltar a trabalhar depois por exemplo.
E um grande atrativo obviamente é receber em dólar/euro, mesmo que abaixo do valor do mercado onde a empresa atua, por exemplo em Nova Iorque 80k dólares anuais está longe do topo da "cadeia alimentar", e já seria um salário mais que excelente pra alguém morando no Brasil, enquanto é "apertado" pra quem mora por lá.

2. O que leva uma empresa a buscar desenvolvedores remotos?
A grande maioria, mão de obra barata, simples assim, só ver o exemplo aí de Londres quanto custa um Dev local e quanto oferecem pra um Dev remoto, o motivo é esse eu diria que em 99% das vezes, seja só o salário como também custos que eles tem que arcar de acordo com as leis trabalhistas locais.

3. O inglês é um diferencial que deveria potencializar a remuneração? Por quê?
Depende, inglês para leitura e escrita básica/intermediário eu acho que é o mínimo e não diferencial. Fluência pra se comunicar com equipes internacionais, assumir responsabilidades com isso, ou mesmo atender a clientes que você precisa do inglês pra conversação, aí sim acredito ser um diferencial que deve refletir na remuneração. Não adianta você trabalhar em uma agência que atende só clientes locais e não tem planos de atender clientes internacionais, e querer pleitear aumento só porque tem inglês fluente, não faz diferença alguma pra eles.

4. No contexto de trabalho remoto internacional, nós merecemos receber mais do que no mercado local? Por quê?
Mais uma vez, depende. Em São Paulo você vai encontrar diversas oportunidades com remuneração acima de R$10k, mas trabalhando remoto pra uma empresa de Portugal por exemplo, você dificilmente vai conseguir ganhar mais de €2k líquido, que seriam ~R$8k. Agora imagina o outro exemplo que dei pra uma empresa de uma área cara dos EUA, ou até Alemanha, Holanda, que pagam salários de €45-60k anuais (~R$17-22.5k mensais), aí sim você tem a possibilidade de atingir patamares bem mais altos de remuneração sem a necessidade de ser um Gerente, Tech Lead, Arquiteto, etc. Merecer ninguém merece nada, mas se correr atrás tem como chegar lá.

**Comment by henriquegibi** [comment]
Olá pessoal. Gostaria de agregar minha opinião.

Parece exploração? Pois agradeça de ser assim. Graças à isso temos nossas montadoras, uma quantidade de indústrias, e exportamos alguns pouquíssimos produtos industrializados: produzir aqui é mais barato. Aqui é mais barato que lá.

Além disso não era uma vaga internacional, era remoto. Se eu tivesse condições de atender àquela vaga, eu me candidataria, aceitaria e iria trabalhar de lá do Paraguay (Ciudad del Este) onde a mensalidade de uma faculdade de medicina gira em volta de 700 reais e 1100. Minha esposa adoraria cursar medicina, é o sonho dela.

Uma vaga nestas condições atende meus objetivos, eu é que não atendo aos requisitos da vaga. Para quem tem por objetivo comprar um Maserati, esta vaga pode não atendê-lo também. Ou pode. Não me dei ao trabalho de fazer as contas.

Fora isso, não vou repetir o que já foi dito, e gostei do wall-of-text do amigo alí, e das demais opiniões.

**Edit: 22/08/2019**

A vaga não é internacional. Não sei porque chegaram à esta conclusão. Tem escritório em Londres? Beleza, mas a vaga continua sendo remota, e não internacional.

Isso [aqui](https://www.linkedin.com/posts/rom%C3%A1rio-da-silva_a-holden-ti-atendendo-importante-consultoria-activity-6570001494249091073-Yr9Q) sim é uma vaga internacional, quando a empresa banca a sua mudança para outro país.

A propósito, eu já ocupei uma vaga internacional, fui realocado em outro país por 1 ano e 4 meses. E a vaga em questão, na moral, não é internacional.

**Comment by kazzkiq** [comment]
Minha opinião é que R$8k pra um trampo internacional (PJ) é baixo.

Além dos custos com empresa, contador e impostos, é importante lembrar como nossa moeda é desvalorizada em comparação à Dolar/Euro/Libras.

Hoje **8.000,00 BRL** equivalem a **1.800,00 EUR**. Pelas vagas que eu vejo por aí, isso é metade do que as empresas pagam em países como Holanda, Alemanha e Reino Unido. É um valor **muito abaixo** da média de mercado deles.

Se você tem um inglês afiado e experiência com as tecnologias pedidas na vaga, eu sinceramente não vejo porque não negociar um valor mais próximo da média salarial desses países, visto que trabalho remoto vem se tornando cada vez mais difundido nas empresas e não existe nenhum motivo pelo qual um dev remoto valha menos atualmente.

Então pra resumir: Desses 8k você estaria "desperdiçando" uma grande quantia com impostos e gastos de empresa/infra/contador, se propondo a mudar seu fuso-horário para se adequar à um pais estrangeiro (o que eventualmente pode impactar a saúde), assumindo risco de calotes e cancelamento de contrato a qualquer momento, e tudo isso pra receber a metade do que um dev ganha em média lá na Europa (3-4k EUR).

Pode ser uma ótima alternativa pra quem não ganha um salário nessa faixa hoje no Brasil? Com certeza! Mas pra quem tá com uma segurança financeira maior, eu recomendo negociar ou procurar uma outra vaga internacional com um valor mais interessante.

**Comment by BrunoSette** [comment]
Os salários são regidos pela lei da oferta e procura. Não acredito em valores justos ou injustos. 

Se acha que está ganhando pouco, procure outra vaga, se acha que tá ganhando muito segura! 

Simples assim.

**Comment by wdss93** [comment]
Um salário que paga no **máximo** 8k como **PJ** no Brasil já é um valor que muitos não aceitam. Se tratando de uma empresa estrangeira que, provavelmente, ganhará em libra esterlina, a remuneração é bem baixa. Além do mais querendo desenvolvedor pleno/sênior com inglês fluente.

**Comment by bbertolani** [comment]
Pessoal acho relevante falar de todas as possibilidades, eu não sou da área de frontend mas gosto muito desse grupo e achei melhor compartilhar minha história.
No meu caso eu comecei a trabalhar remotamente a pouco mais de 4 anos e sempre girando em torno de 2K USD nos primeiros meses e depois foi para 3K ou +, isso trabalhando full time.
2 anos após eu começar a trabalhar achei um fixo interessante no Brasil e avise a empresa do exterior que iniciaria nesse novo trabalho e permaneceria como freelance, eles aceitaram e continuei a trabalhar nas horas vagas, na minhas ferias fui convidado a visitar a empresa no Canadá e hoje mesmo estou em Ottawa pela 3x.
Venho para cá por cerca de 1 mês, e a empresa está suportando todo o meu processo de migração.
Então assim, eu trabalhei muito, mas vejo que depois de quase 5 anos me parece que os resultados estão aparecendo. Acredito que tudo depende do objetivos de cada um.

**Comment by walteraa** [comment]
Pessoal,

Eu não iria falar nada mas tenho que pontuar algumas coisas:

- Como disseram, não tem nada demais em oferecer job PJ up to 8K BRL, livre mercado, aceita quem quer. Seniores vão aceitar? Muito provavelmente não. Plenos vão aceitar? Muito provavelmente sim.
- Dizer que está acima da média nacional já é demais: Eu como sênior tiro mais que isso Líquido com CLT + todas suas vantagens(plano de saúde, adicional de férias, férias remuneradas de 30 dais, 13o, etc...), full remoto: Trabalho alocado numa empresa dos EUA mas contratado por uma empresa BR.
- O único problema que eu vejo na vaga é que abrindo o link da descrição dela, me deparo com isso:
> Competitive pay - we believe in fair compensation for great work and our ambition is to find and keep talented people for many years to come.

Bem, não é um salário competitivo sequer pros padrões brasileiros, quisar pros padrões UK, mas como disseram, não é uma vaga internacional e isso nos leva ao último ponto:

- A empresa é brasileira? Não. O time é full Brasileiro? Não. Os clientes são brasileiros na sua totalidade? Não. Posso ser considerado pra vaga falando apenas português? Certamente que não. Conslusão: A vaga é sim uma vaga internacional. Ponto.

**Comment by eliseumds** [comment]
Eu recomendo focar no inglês, sair do Brasil por um tempo e depois, se quiser, retornar com propostas salariais muito maiores. Ou continuar fora. Ou conseguir vagas remotas pagando muito mais por causa da experiência internacional. Eu falo em coisa de 15k USD pra cima (existem salários bem altos de empresas no Brasil tbm).

A grande maioria das empresas, como o @guibfo disse, procuram trabalhadores remotos pra pagarem menos.

**Comment by rodrigodlima** [comment]
Olá pessoal, achei muito pertinente essa thread, tenho uma oferta para trabalhar remotamente para uma empresa na Escócia. 
Minha dúvida está um pouco fora do que a thread propõe, mas acho que aqui posso ter a opinião de algumas pessoas que já passaram por isso.
Como seria a contratação por uma empresa internacional? Se for como PJ, a minha empresa registrada aqui no Brasil atenderia? Ou teria que abrir uma empresa do país de origem da empresa? Caso a contratação seja como CLT por exemplo, seria pelas normas Brasileiras ou teria opção de ser pelas normas trabalhistas do país de origem? Sobre receber em dólar, euro ou libras, teria que abrir uma conta internacional ou algum banco Brasileiro aceitaria?

**Comment by annebione** [comment]
Essa questão é mais simples do que parece.

Se você é senior aqui no Brasil e tá ganhando menos ou igual a junior no país de origem da empresa é exploração sim. Dá pra aceitar alguns salários de junior, ou junior-pleno nos EUA ou Europa da vida pra um desenvolvedor Senior aqui do Brasil. Há diferenças culturais de trabalho, e diferenças de idioma. Avalio que é o clássico caso do "Cada um cede um pouquinho e todo mundo ganha". 

Senior brasileiro ganhando menos que junior na europa/EUA, porém, é exploração de mão de obra barata latina sim. Simples e direto. Você pode aceitar se quiser, problema teu, mas tenha consciência do que você tá sendo - mão de obra barata e desvalorizada, consequentemente, desvalorizando seu trabalho e dos seus colegas do mesmo país - e respeite quem não quer se sujeitar a isso.

Em tempo, quem quiser ler mais sobre essa linha de raciocínio, recomendo MUITO o artigo do Basecamp (que tem uma equipe remota do mundo todo) explicando como eles pagam os desenvolvedores deles:
https://m.signalvnoise.com/how-we-pay-people-at-basecamp/

**Comment by lucasferreiralimax** [comment]
Muito interessante esse tópico sobre trabalhos internacionais e seus valores, mas uma dúvida pertinente a forma de contratação sempre vai ter que ser como exportação de trabalho de sua empresa? No caso não teria nenhuma outra forma de contratação para manter ou garantir algum tipo de contrato e confiança entre a prestação de serviço remota feita internacionalmente?

**Reply by gabriel-hahn** [reply]
Opa @lucasferreiralimax, normalmente é tudo baseado no contrato, dai depende da empresa e do acordo de vocês dois, todos os detalhes estão no contrato (até mesmo os detalhes como a quantidade de dias de férias, que pode variar, ou se a empresa paga horas extras). Não sei informar se existe alguma outra maneira além desse 'estilo PJ' (tanto remotamente ou presencial), mas creio que não 🤔

**Comment by kunzeericchristopher-png** [comment]
Você está procurando por hackers confiáveis, rápidos e profissionais? Esta equipe de hackers provou ser competente e me trouxe tranquilidade. Envie uma mensagem para eles em:

e-mail: NOBLEWEBHACKERS@GMAIL.COM

Website: https://noblewebhackers.vercel.app/

Lista de seus serviços:
Recuperação de criptomoedas de plataformas falsas/Negociação Binária
Hack de redes sociais
Hack de portal de carteira de motorista
Hack de score de crédito
Hack de notas universitárias
Hack de senha de iPhone
Hack de cartões de crédito/cartões em branco
Hack de localização
Hack de conta bancária, entre muitos outros, todos com garantia de 100% de sucesso...
